.gitignore

```diff
@@ -1,10 +1,18 @@
+# dependencies
 /node_modules
+
+# local
 .DS_Store
+
+# production
 /docs
 !/docs/README.md
 /dist
-.env
 bundleStats.html
-local_temp_server_files/
-.vscode/
-coverage/
+/localDevPodStorage
+/local_temp_server_files
```

Comment:
What's the difference between `localDevPodStorage` and `local_temp_server_files`?